src/client/datascience/kernel-launcher/kernelDaemon.ts

```diff
@@ -107,6 +107,10 @@ export class PythonKernelDaemon extends BasePythonDaemon implements IPythonKerne
         );
         let stdErr = '';
         this.connection.onNotification(KernelDiedNotification, (output) => {
+            // If we have requested for kernel to be killed, don't raise kernel died error.
+            if (this.killed) {
```

Comment:
We were unnecessarily throwing & logging errors when we were manually killing he kernels.

Reply:
Is this the error code -9 warning?